tests/x11/exiv2.pm

```diff
@@ -0,0 +1,106 @@
+        expected_output_exiv2_ptinfo_description      => "Exif.Image.ImageDescription                  Ascii      19  Test caption label",
+        expected_output_exiv2_ptinfo_data             => "Exif.Image.DateTime                          Ascii      20  2019:02:01 15:44:21",
+        expected_output_exiv2_ptinfo_artist           => "Exif.Image.Artist                            Ascii       5  SUSE",
+        expected_output_exiv2_ptinfo_copyright        => "Exif.Image.Copyright                         Ascii       9  SUSE Inc",
+        expected_output_exiv2_ptinfo_exiftag          => "Exif.Image.ExifTag                           Long        1  288",
+        expected_output_exiv2_ptinfo_fnumber          => "Exif.Photo.FNumber                           SRational   1  F1.1",
+        expected_output_exiv2_ptinfo_brightness       => "Exif.Photo.BrightnessValue                   SRational   1  1.1",
+        expected_output_exiv2_ptinfo_max_aperture     => "Exif.Photo.MaxApertureValue                  SRational   1  F1.3",
+        expected_output_exiv2_ptinfo_lightsource      => "Exif.Photo.LightSource                       SLong       1  Daylight",
+        expected_output_exiv2_ptinfo_focal_length     => "Exif.Photo.FocalLength                       SRational   1  50.0 mm",
+        expected_output_exiv2_ptinfo_user_comment     => "Exif.Photo.UserComment                       Undefined  26  Test caption label",
+        expected_output_exiv2_ptinfo_subject_distance => "Exif.Photo.SubjectDistanceRange              SLong       1  Close view",
+        expected_output_exiv2_ptinfo_jpeg_format      => "Exif.Thumbnail.JPEGInterchangeFormat         Long        1  818",
+    );
+
+    my $output_exiv2_info = script_output "exiv2 test.jpg";
+    my @exif_infos        = keys %exif_expected_output;
+
+    for my $exif_info (@exif_infos) {
+        die "Missing exiv2 info. Expected: /$exif_expected_output{$exif_info}/ \nGot: /$output_exiv2_info/"
+          unless $output_exiv2_info =~ m/$exif_expected_output{$exif_info}/;
+    }
+
+    my @exif_ptinfos        = keys %exif_expected_ptoutput;
+    my $output_exiv2_ptinfo = script_output "exiv2 -pt test.jpg";
+
+    for my $exif_ptinfo (@exif_ptinfos) {
+        die "Missing exiv2 info. Expected: /$exif_expected_ptoutput{$exif_ptinfo}/ \nGot: /$output_exiv2_ptinfo/"
+          unless $output_exiv2_ptinfo =~ m/$exif_expected_ptoutput{$exif_ptinfo}/;
+    }
+}
+
+sub run {
+    select_console "x11";
+    x11_start_program('xterm');
+
+    become_root;
+    pkcon_quit;
+    zypper_call "in exiv2";
+    type_string "exit\n";
+
+    assert_script_run "wget --quiet " . data_url('exiv2/test.jpg') . " -O test.jpg";
+
+    exiv2_info_test;
+
+    assert_script_run "exiv2 rename test.jpg";
+    assert_script_run "ls -ld 20190201_154421.jpg";
+    script_run "eog 20190201_154421.jpg", 0;
+    assert_screen "exiv2_rename_test";
+    send_key 'alt-f4';
+
+    assert_script_run "exiv2 -ep1 20190201_154421.jpg";
```

Comment:
come comments might be useful here to see what is it doing

Reply:
ok, thank you for the suggestions